### PR TITLE
Ensure up-to-date dependencies for Capstone Python bindings

### PR DESCRIPTION
**Pull request by kabeor**
The `base-builder-python` docker image installs `setuptools v42.0`. This is very likely the reason why Capstone's `setup.py` fails.

@DavidKorczynski Is there any reason why it is set to this (very old) version? The [docs](https://google.github.io/oss-fuzz/getting-started/new-project-guide/python-lang/#dockerfile) recommend this one.

### CODE REVIEW

**Comment by DaveLak** [comment]
> Is there any reason why it is set to this (very old) version? The [docs](https://google.github.io/oss-fuzz/getting-started/new-project-guide/python-lang/#dockerfile) recommend this one.

I'm a different David than the one you asked but I think I can answer the question:

Likely because upgrading it impacts every integrated Python project and it's somewhat time consuming to:
- verify the change doesn't introduce new build failures.
- the debug & patch any new failures that do get introduced by the upgrade.

I've been keen on seeing this bumped as well, but last I checked a few months ago, there were a few projects that required patches to prevent new failures.

I was planning on trying to upgrade `setuptools` and `pyinstaller` after upgrading Python to 3.10 in #12027 (if I'm able to get that landed) since it will enable the removal of a lot of adhoc workarounds that many Python projects are currently employing which would hopefully make a dependency upgrade simpler to manage.

File: projects/capstone/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder-python
 RUN apt-get update && apt-get install -y make cmake
-RUN git clone --depth 1 --branch v5 https://github.com/aquynh/capstone.git capstonev5
-RUN git clone --depth 1 --branch next https://github.com/aquynh/capstone.git capstonenext
+RUN pip3 install --upgrade setuptools build wheel
```
Comment (DaveLak) [inline projects/capstone/Dockerfile:19]: You might want to consider pinning these to a known stable version. It'll mean manual version bumps, but might prevent or allow easier debugging of unexpected build failures in the future.

Comment (Rot127) [reply projects/capstone/Dockerfile]: Good point, we will take care of it.

**Comment by kabeor** [comment]
@DaveLak Thank you very much for your help, I guess we finally solved this by upgrading pip.

**Comment by kabeor** [comment]
@DavidKorczynski Hi, it's ready to be merged, thanks:)

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**